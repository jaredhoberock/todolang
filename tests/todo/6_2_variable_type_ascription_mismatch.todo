var x: String = 13;
print x;
