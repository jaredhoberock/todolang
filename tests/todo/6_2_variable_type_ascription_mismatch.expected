[0m[1m[38;5;9merror[0m[1m: Type mismatch: expected 'String', found 'Number'[0m
  [0m[34mâ”Œâ”€[0m tests/todo/6_2_variable_type_ascription_mismatch.todo:1:5
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m var [0m[31mx: String = 13[0m;
  [0m[34mâ”‚[0m     [0m[31m^^^^^^^^^^^^^^[0m


